HACKING.md

```diff
@@ -4,14 +4,39 @@ Run `./tool/build.sh` once. This will download the correct version of the Replic
 
 You won't need to do this again unless you want to pick up a newer version of that library.
 
-Get the [`repl` command line](https://github.com/rocicorp/replicache/releases) and start a dev server:
+# Testing
+
+Unit testing is accomplished mostly in the normal Flutter fashion:
+
+```
+flutter test
+```
+
+By default, unit tests run against a mock replicache-client which
+returns a pre-recorded set of responses. See [sync_replay.json](test/sync_replay.json).
+
+To update the recording, modify `replicache_test.dart` thusly:
+
+```
+-    recordPath = './sync_replay.json';
+-    // await useReplay('./sync_replay.json');
++    // recordPath = './sync_replay.json';
++    await useReplay('./sync_replay.json');
+```
+
+then start a replicache-client test server:
+
+```
+go run ./build/replicache-client/cmd/test_server/main.go &
+```
+
+then re-run the tests:
 
 ```
-repl --db=/tmp/wherever serve
+flutter test
 ```
 
-Once you've done this, you can run the sample apps (in `sample` directory) and they will pick up Dart changes
-in the library as you make them.
+**Don't forget to undo the change to `replicache_test.dart` before landing!**
```

Comment:
I'm doing this with an env variable in my current branch. Makes it cleaner

Reply:
Thanks! Just update this when it lands.